resources/zib/zib-Procedure-request.xml

```diff
@@ -109,20 +115,7 @@
         <rules value="open" />
       </slicing>
       <short value="ProcedureStartDate / ProcedureEndDate" />

```

Comment:
This `.short` element should be removed (compare with `Procedure.performed[x]` in the zib-Procedure-event profile).

Reply:
Removed